themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,181 @@
+
+## Azure App Service
+
+Azure App Service is a more fully-featured offering for hosting applications, focusing more on WebApps and API services. It started life as several differing offerings (Azure Websites, Azure Mobile Service & Azure BizTalk) before being bundled into a single service in March 2015. Its origins as multiple solutions means it has developed tight integrations with other Azure solutions and offers many value-add features depending on your use case.
+
+It’s important to note that because of the myriad features Azure App Service offers, running containers is only a small part of what you can do with it.
+
+With that in mind, let’s take a look at where it shines with regards to running containers.
+
+### What’s it good for?
+
+Azure App Services is designed to be a one stop shop for your web application needs, so it’s highly opinionated and pushes you in a certain direction
+
+#### Integrations
+
+Azure App Service has many integrations with Azure. As an example, if you need to add a CDN to your web service, it’s extremely straightforward to do so with Azure CDN.
+
+This tight integration with Azure’s many other services makes it likely to meet the needs of your application and extend it as needed, introducing common architecture components like message queues and databases.
+
+#### Scaling & Availability
+
+Azure App Service has a myriad of available scaling capabilities built out of the box. Not only can you scale the actual container you’ve deployed (while also simultaneously placing pricing caps on your service) but you can also scale related resources if necessary, which fits well with the tight integration we mentioned previously.
+
+App service scaling for your container can either scale up (ie: increase the amount of resources associated with your container) or scale out (ie: have more containers behind your load balancer).
+
+This flexibility in scaling means you can be safe to assume your application will meet spiking traffic requirements and service your users’ needs.
+
+### What’s not great about it?
+
+#### Opinionated
+
+Azure App Service is highly opinionated. It’s tight integrations with other Azure services mean that you’re expected to follow established patterns for achieving your goals.
+
+This even extends to the allowed languages that Azure App Service supports - if you’re developing your application in an unsupported language, you can’t use the service.
+
+This opinionated implementation means that the initial learning curve can be steep. The origins of app services means the product has grown in complexity along with its opinionated implementation, and as a result it can be overwhelming when getting started.
```

Comment:
```suggestion
This opinionated implementation means that the initial learning curve can be steep. The origins of app services means the product has grown in complexity along with its opinionated implementation, and as a result, it can be overwhelming when getting started.
```